spaces/S000170/properties/P000200.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000170
+property: P000200
+value: false
+refs:
+- zb: "1044.55001"
+  name: Algebraic Topology (Hatcher)
+---
+
+Theorem 1.7. on page 29 of {{zb:1044.55001}} calculates the fundamental group of $S^1$ and since it's not trivial, there is a continuous map $S^1\toS^1$ not homotopic to a constant map.
```

Comment:
```suggestion
Theorem 1.7.on page 29 of {{zb:1044.55001}} calculates the fundamental group of $S^1$ as $\mathbb Z$. Since it's not trivial, there is a continuous map $S^1\toS^1$ not homotopic to a constant map.
```

Reply:
You can accept the whole suggestion with the "comment suggestion" button, or manually edit the changes you want into a new commit.